lib/muffin_man/muffin_logger.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "logger"
+module MuffinMan
+  class MuffinLogger
+    def self.logger
+      defined?(Rails) ? Rails.logger : Logger.new($stdout)
```

Comment:
Instead of making this gem aware of rails, we can add support for the application using the gem to set a logger. Like how Rails supports doing `Rails.logger = Logger.new`. To do this you can add `:logger` as an `attr_accessor` like we have done for `:configuration` on [this line](https://github.com/patterninc/muffin_man/blob/master/lib/muffin_man.rb#L37) 

```ruby
  class << self
    attr_accessor :configuration, :logger
  end
```

Then we can use the logger set by the application if one is set with 
```ruby
MuffinMan.logger.info "a message for the logs"
```

If no logger is set, I wonder if we default to `Logger.new("/dev/null")` so that nothing actually gets logged but we can still write code assuming `MuffinMan.logger` is always set.